editor/scene_tree_dock.cpp

```diff
@@ -1157,6 +1157,19 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 	}
 }
 
+void SceneTreeDock::_property_selected(int p_idx) {
+	ERR_FAIL_NULL(property_drop_node);
+	_perform_property_drop(property_drop_node, menu_properties->get_item_metadata(p_idx), ResourceLoader::load(resource_drop_path));
+	property_drop_node = nullptr;
+}
+
+void SceneTreeDock::_perform_property_drop(Node *p_node, String p_property, RES p_res) {
+	editor_data->get_undo_redo().create_action(TTR("Set") + " " + p_property);
```

Comment:
I think it would be better to use `vformat(TTR("Set %s"), p_property)` instead, as the property might not come to the right of the "set" word in other languages.

Reply:
I copied this from the inspector:
https://github.com/godotengine/godot/blob/e1547798d661e972e06860a33b92f7cbd6ec69d0/editor/editor_inspector.cpp#L2264

Reply:
Oh. I guess you could change it there too? 
Even if it's unrelated to this PR it's a pretty small change that should not hurt the PR backlog that much.